src/pybind/mgr/dashboard/frontend/cypress/e2e/cluster/hosts.e2e-spec.ts

```diff
@@ -31,4 +31,15 @@ describe('Hosts page', () => {
       hosts.check_for_host();
     });
   });
+
+  describe('should have all host details', () => {
+    it('should have hostname'),
+      () => {
+        cy.get('[data-testid="hostname"]').should('not.be.empty');
+      };
```

Comment:
shouldn't it be like this?


```suggestion
    it('should have hostname', () => {
        cy.get('[data-testid="hostname"]').should('not.be.empty');
      });
```


I saw these 2 tests were in [Pending state ](https://jenkins.ceph.com/job/ceph-dashboard-pull-requests/21358/console)but never ran
```
should have all host details
      - should have hostname
      - should have IP address
```

Also does it need cephadm for the ip address or you wll get it even if you are not on cephadm? if its cephadm dependant we need to move the tests under the `e2e/orchestrator`, Maybe inside `08-hosts.e2e-spec.ts`.

Reply:
Yes, moved to adm tests